File: Contributors/BryanYunis/solutions/intersecting_lines.py
def intersecting_lines(p, q):
    intersections = 0
    for i in range(len(p)):
        for j in range(len(p) - i  - 1):
            if (q[j + 1] < q[i]):
                intersections = intersections + 1
    return intersections

print(intersecting_lines([1, 2, 3], [1, 2, 3]))

File: Contributors/BryanYunis/solutions/intersectingLines.js
function intersectingLines(p, q) {
    let intersections = 0
    for (let i =0; i < p.length; i++){
        for (let j = i + 1; j < q.length; j++) {
            if (q[j] < q[i]) {
                intersections++
            }
        }
    }
    return intersections
}

console.log(intersectingLines([1, 2, 3], [2, 1, 3]))